src/devices/xiaomi.ts

```diff
@@ -814,8 +814,16 @@ const definitions: Definition[] = [
         model: 'ZNXDD01LM',
         vendor: 'Xiaomi',
         description: 'Aqara ceiling light L1-350',
-        extend: xiaomiExtend.light_onoff_brightness_colortemp({disableEffect: true, colorTempRange: [153, 370]}),
```

Comment:
You can keep using extend, use this instead: `extend.light_onoff_brightness_colortemp_color({fromZigbee: [fz.aqara_opple]})`